### PR TITLE
Copy-edit Bolts

### PR DESCRIPTION
**Pull request by toadlyBroodle**
Update stylesheet guidelines and apply updates to BOLT 0, 1, 2.
Complete first pass copy-edit on BOLT 3.

### CODE REVIEW

File: 03-transactions.md [hunk 1]
```diff
@@ -285,165 +291,171 @@ The fee calculation for both commitment transactions and HTLC
...
-secrets are known (i.e., `localkey`, `local_htlckey` and `local_delayedkey` only):
+The corresponding private keys can be similarly derived, if the basepoint
+secrets are known (i.e. `localkey`, `local_htlckey`, and `local_delayedkey` only):
 
     secretkey = basepoint_secret + SHA256(per_commitment_point || basepoint)
 
 ### `revocationkey` Derivation
 
-The `revocationkey` is a blinded key: when a node wishes to create a new
+The `revocationkey` is a blinded key: when a local node wishes to create a new
```
Comment (rustyrussell) [inline 03-transactions.md]: I think I prefer "the local node" and "the remote node" here, actually.

**Comment by toadlyBroodle** [comment]
Good calls, done and done.

**Comment by rustyrussell** [comment]
Will commit once I have a :+1: from any other pair of willing eyeballs :)

**Comment by toadlyBroodle** [comment]
So can I resolve these conflicts or leave that to you @rustyrussell? Last time I resolved the conflicts on github and it seemed to do some funny things. What's the best way to resolve these conflicts in my local repo using command line?

**Comment by rustyrussell** [comment]
I'll do it, since I produced the conflicts :)

I'm just waiting for @shannona who said he'd review within 24 hours.

**Comment by shannona** [comment]
Yep, I'll hit it before lunch.

File: 02-peer-protocol.md [hunk 2]
```diff
@@ -211,8 +212,8 @@ The sender:
 avoid double-spending of the funding transaction.
 
 The receiver:
-  - if the `chain_hash` value within the `open_channel` message is set to a hash
- of a chain unknown to the receiver:
+  - if the `chain_hash` value, within the `open_channel`, message is set to a hash
+ of a chain which is unknown to the receiver:
```
Comment (shannona) [inline 02-peer-protocol.md]: This "which is" should be "that is" because the clause that follows is necessary to the sentence.

Comment (rustyrussell) [reply 02-peer-protocol.md]: OK, I'll fix this nit, rebase and merge now!

**Pull request merged**